.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,20 @@
+name: Deployment
+on:
+  push:
+    branches-ignore: ["*"]
+    tags: ["*"]
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags') && github.repository == 'sprockets/sprockets.http'
+    container: python:3.9-alpine
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v1
+      - name: Build package
+        run: python3 setup.py sdist
```

Comment:
Just to note, it was building wheel prior

Reply:
Added wheel building